better-restaurant/src/components/SigninComponent.js

```diff
@@ -2,13 +2,108 @@ import React from 'react';
 import styled from 'styled-components'
 
 const Container = styled.div`
-  
+  padding-left: 20px;
 `
+//figma.com
+
+const LoginTitle = styled.div`
+	margin-height: 25vh;
+	font-family: Inter;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 36px;
+	line-height: 22px;
+	padding-left: 22vw;
+
+	color: #1A61BD;
+`
+
+const MainBox = styled.div`
+	width: 72vw;
+	height: 76vh;
+	background: #FFFFFF;
+	border: 3px solid #E9E9EF;
+	box-sizing: border-box;
+	box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+`
+
+const Index = styled.div`
+	width: 102px;
+	height: 25px;
+	font-size: 20px;
+	line-height: 22px;
+
+	display: flex;
+	align-items: center;
+
+	color: #000000;
+`
+const Input = styled.div`
+	width: 47vw;
+	height: 64px;
+
+	background: #E5E5E5;
+	border-radius: 10px;
+`
+
+
+
+const GrayIndex3 = styled.div`
+	width: 126px;
+	height: 34px;
+	font-size: 20px;
+	line-height: 22px;
+
+	color: #85859A;
+`
+const SignUpText = styled.div`
+	font-size: 25px;
+
+	display: flex;
+	align-items: center;
+	text-align: center;
+
+	color: #4286F5;
+`
+const BlueIndex2 = styled.div`
+	width: 47vw;
+	height: 8vh;
+
+	background: #4286F5;
+	border: 1px solid #437DD1;
+	box-sizing: border-box;
+	border-radius: 10px;
+`
+
+const WhiteIndex = styled.div`
+	font-weight: bold;
+	font-size: 36px;
+
+	color: #FFFFFF;
+`
+
+const Checkbox = styled.input`
+	background-color: blue;
+	color: blue;
+`
+
 
 export default function SigninComponent(){
   return (
     <Container>
-      Signin
+			<MainBox>
+				<LoginTitle>부대계정로그인</LoginTitle>
+				<Index>ID</Index>
+				<Input></Input>
+				<Index>PASSWORD</Index>
+				<Input></Input>
+				<Checkbox type="checkbox"/>
+				<GrayIndex3>자동 로그인</GrayIndex3>
```

Comment:
GrayIndex3 처럼 알 수 없는 숫자를 붙이는건 좋지 않음.
1, 2, 3 순서라고 해도 각각이 무엇을 의미하는지 알아볼 수 없으니 이름을 따로 작성하도록.

자동 로그인은 AutoLogin이라고 표현할 수 있으니, 코드를 다음과 같은 형태로 바꾸는 것이 좋을듯.

```suggestion
<AutoLogin>
  <Checkbox type="checkbox"/>
  <Label>자동 로그인</Label>
</AutoLogin>
```
"input의 종류가 뭐던 그 옆에 두는 설명"은 '**label**' 이라는 표현을 통용적으로 쓴다는 것을 알아두면 좋을듯.